### DISCUSSION TITLE
Error :: Add-PnPCustomAction : Attempted to perform an unauthorized operation

### DISCUSSION THREAD

**Discussion by fofawowe** [pnp/powershell#2575] [Q&A]
### Discussed in https://github.com/pnp/powershell/discussions/2369

<div type='discussions-op-text'>

<sup>Originally posted by **MAP-Git** September 21, 2022</sup>
Getting this error: "Add-PnPCustomAction : Attempted to perform an unauthorized operation." User has Admin level access on SharePoint Online. PnP PowerShell throws error. Please advice solution here.</div>

**Comment by fksontini** [comment]
Run the following command to disable the security temporarily:
Set-PnPSite -NoScriptSite $false

After executing your script, remember to re-enable the security by setting it back to true.
